interface/ceed-basis.c

```diff
@@ -1314,21 +1310,13 @@ int CeedSymmetricSchurDecomposition(Ceed ceed, CeedScalar *mat,
     }
     CeedScalar norm = sqrt(v[i]*v[i] + sigma); // norm of v[i:n-1]
     CeedScalar R_ii = -copysign(norm, v[i]);
-    v[i] -= R_ii;
```

Comment:
I presume deleting this line was unintentional. Fixed in 3ceb3616063706ae53d944dabb70706b67b16aa8 and all tests pass locally.

Reply:
Hmm, yes. That was an unintentional deletion.